osmosis-1/osmosis-1.assetlist.json

```diff
@@ -1381,6 +1381,33 @@
         "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/injective/images/inj.svg"
       },
       "coingecko_id": "injective-protocol"
+    },
+    {
```

Comment:
Please remove from osmosis-1..assetlist.json. It should only be on the Frontier doc for now.